mftf/2.3/reporting.md

```diff
@@ -0,0 +1,368 @@
+---
+---
+
+# Reporting
+
+The Magento Functional Testing Framework provides two types of reporting:
+
+- In CLI, when you run [`mftf`][mftf] or [`codecept`][codecept] commands.
+- In a web browser, using [Allure Framework][] after test run was complete.
+
+All the reporting artifacts are stored at the `dev/tests/acceptance/tests/_output` in the Magento root directory.
+The directory contains:
+
+- `allure-report/` that is a directory generated and served by the Allure Framework.
+- `failed` that is a text file containing relative paths to failed tests after the last test run.
+  The paths are relative to `dev/tests/acceptance/`.
+- `.html` and `.png` files that are screenshots of fails in HTML and PNG formats.
+  The files are appended after each new run.
```

Comment:
Are these two notes about appended files and cleanup for all of these options? Looking at the format, they may generate as part of the .html and .png file bullet point. May need to move to a new line, remove indent. I would check the rake build.

Reply:
No, only for the last option. It is a part of "one sentence - on line" syntax for lists in Markdown.